setup.py

```diff
@@ -49,6 +49,7 @@
         'Topic :: System :: Filesystems',
     ],
     install_requires=['apache-libcloud',
+                      'Deprecated >= 1.2.10',
```

Comment:
instead of introducing yet another dependency, i'd suggest just update all the callers of the deprecated functions, and then drop them.

Reply:
what is the problem with introducing new dependency which is useful?

Reply:
> i'd suggest just update all the callers of the deprecated functions, and then drop them.

Reply:
@tchaikov I've dropped the dependency.